src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutBase.cs

```diff
@@ -297,6 +300,7 @@ private protected virtual void ShowAtCore(FrameworkElement placementTarget, Flyo
 			}
 
 			Target = placementTarget;
+			XamlRoot = placementTarget?.XamlRoot;
```

Comment:
I believe this is the only common point of entry that opens the flyout and triggers `Open(ing|ed)` events.
... Right?

Reply:
Well nothing else invokes Opened/ing, IsOpened.set.